open-craft/opencraft
# Run an extra custom playbook on all appservers.

A default custom playbook will run on all new Open edX appservers that installs some common services like Consul, Prometheus Node Exporters, Filebeat, etc..

To test this:

1. Go on stage and set proper stage settings for Prometheus Node Exporters & Consul according to the new settings that were added in this PR.
2. Redeploy Ocim stage.
3. Deploy a new appserver.
4. Check that its configuration settings show the new variables, plus the consul service definition.
5. Check that the custom playbook is the one being run.
6. When the playbook finishes running, check on the server that the node exporter, consul and filebeat are all installed successfully.
